src/sources.cmake

```diff
@@ -76,34 +81,10 @@ src/ResultWriter/FreeSurfaceWriter.cpp
 src/ResultWriter/EnergyOutput.cpp
 
 # Fortran:
```

Comment:
Could you, please, delete this line while you will be doing a merge with the master?